### DISCUSSION TITLE
Can't Stream Games... ?

### DISCUSSION THREAD

**Discussion by RedMonster2** [nfriedly/nodeunblocker.com#39] [Q&A]
I've tried to run a couple of .io games via the unblocker, it runs very  well, very smoothly! but when i try to join the game (or if the site tries to join on load) I can't access it, i just get stuck on the loading screen, for example, slither.io, when i push play, i'm just stuck with the arrows spinning in the middle of the screen.

Can this be fixed?

If not, are their other unblockers that can handle such?

thank you

**Comment by nfriedly** [comment]
Hey, sorry I missed this when you first wrote it. There were some updates a couple of years ago to add websockets support, which fixed some .io games. However, some games are probably never going to work via a proxy.
